packages/graphhopper/graphhopper.test.ts

```diff
@@ -1,14 +1,25 @@
 import { GraphHopper } from "./index"
+import { RoutingJSAPIError } from "@routingjs/core"
+import {GraphHopperErrorProps} from "./index"
 import dotenv from 'dotenv'
 dotenv.config()
 
+const assertError = (e: RoutingJSAPIError<GraphHopperErrorProps>)=>{
+    expect(e.properties).toBeDefined()
+    expect(e.properties).toHaveProperty("status_code")
+    expect(e.properties).toHaveProperty("status")
+    expect(e.properties).toHaveProperty("message")
+    expect(e.properties).toHaveProperty("hints")
+    expect(e.properties.hints).toHaveLength(1)
+}
+
 describe("GraphHopper returns responses", () => {
```

Comment:
This goes for all tests of all routers: you are currently mixing test cases that should return successfully and test cases that we expect to throw. Leave the test cases that were present before (that we expect to run successfully, so we can remove `.catch(...)` here) and add additional ones where we except to fail and only add the `.catch()` (we can leave out `.then(...)` since we're not expecting them to succeed anyway). Also, remember to adjust the description of the tests you are expecting to fail to something like "Throws `RoutingJSAPIError`".

Reply:
Oh yes that's a mistake 😅 I was actually making those passing tests as failing tests and then added the catch to check and forgot to create the actual ones